status-bar.ts

```diff
@@ -283,6 +283,7 @@ class StatusBarSprite extends Sprite {
         const sb = this._statusBar;
 
         if (sb) {
+            sb.current
```

Comment:
do we care about ';'? or no? XD

Reply:
Oops typo, good catch! Forgot to push a few small fixes

Reply:
yay